tests/test_modelnet2048.py

```diff
@@ -3,18 +3,23 @@
 import pytest
 import torch
 
+from src.datamodules.common import DataModuleConfig
 from src.datamodules.classification.modelnet2048_module import ModelNet2048DataModule
 
-
+@pytest.mark.slow
 @pytest.mark.parametrize("batch_size", [32, 128])
 def test_modelnet2048(batch_size):
-    data_dir = "data/modelnet2048"
+    config = DataModuleConfig(
+        data_dir="data/modelnet2048",
+        batch_size=batch_size
+    )
+ 
 
-    dm = ModelNet2048DataModule(data_dir=data_dir, batch_size=batch_size)
+    dm = ModelNet2048DataModule(config=config)
     dm.prepare_data()
 
     assert not dm.data_train and not dm.data_test
-    assert Path(data_dir).exists()
+    assert Path(config.data_dir).exists()
 
     dm.setup()
     assert dm.data_train and dm.data_test
```

Comment:
For this one we can also validate that batch.x is empty/null

Reply:
Addressed in #30